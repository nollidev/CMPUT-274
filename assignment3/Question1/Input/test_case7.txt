40 3
W AA AB AC BA BB - - BC - - - BD - - - CA CB CD - - - - - - CE X CG - - - - - - - - - CH - CI
