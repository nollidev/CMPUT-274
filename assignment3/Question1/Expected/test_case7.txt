W -> AA AB AC
AA -> BA BB
BB -> CA CB CD
AB -> BC
BC -> CE X CG
AC -> BD
BD -> CH CI
